packages/console/src/components/Search/index.tsx

```diff
@@ -39,7 +46,7 @@ const Search = ({ defaultValue = '', isClearable = false, onSearch, onClearSearc
         <TextInput
           value={inputValue}
           icon={<SearchIcon className={styles.searchIcon} />}
-          placeholder={t('general.search_placeholder')}
+          placeholder={placeholder ?? t('general.search_placeholder')}
```

Comment:
nit: The usage seems unusual.  Should let the upper scope control the placeholder content? What if some page meant to keep the placeholder empty have to pass an empty string?

Reply:
Cool, updated.